lib/r10k/git/rjgit.rb

```diff
@@ -41,7 +66,6 @@ def self.resolve_version(ver, opts = {})
         repo = opts[:repo] || ::RJGit::Repo.new(opts[:git_dir], is_bare: true)
         ref = repo.commits(ver).first
```

Comment:
In most cases it looks like it will return an empty array but it will return nil if you give it a nil repo so we should probably guard against that.